### PR TITLE
Update repository links to point to main

### PR DESCRIPTION
**Pull request by cipherboy**
`Signed-off-by: Alexander Scheel <alex.scheel@hashicorp.com>`

---

In discussion with @mickael-hc, I was trying to figure out how we got that PR against the old `master` branch. It turns out [the page in question](https://www.vaultproject.io/docs/plugin-portal#community) contains a link to edit this page (at the end of the Community section, ctrl+f for `directly open a pull request with changes to this page`), which is hard-coded (in the page's source) to link to the `master` branch. This doesn't use the correct "Edit this page" link at the bottom of the page (which appears to be a feature of the Vault project hosting website structure, not contained in this repo). 

When editing a page (via the GH web UI) against a specific branch (in this case, `master`) when you "save" your changes and open a pull request, it'll target the branch you were on and not the default branch (makes sense). Hence, we ended up with the PR targeting the `master` branch. 

These appear to be the only self-references to the `master` branch in this repo. 

@hashishaw or @hellobontempo -- could one of you review the changes to the UI directory? Thanks!

### CODE REVIEW

**Review by hashishaw — APPROVED** [review]
Great stuff, thank you!

File: ui/lib/core/stories/alert-inline.md [hunk 1]
```diff
@@ -20,6 +20,6 @@
 **See**
 
 - [Uses of AlertInline](https://github.com/hashicorp/vault/search?l=Handlebars&q=AlertInline)
-- [AlertInline Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/alert-inline.js)
+- [AlertInline Source Code](https://github.com/hashicorp/vault/blob/main/ui/app/components/alert-inline.js)
```
Comment (stevendpclark) [inline ui/lib/core/stories/alert-inline.md:23]: This is a 404

File: ui/lib/core/stories/doc-link.md [hunk 2]
```diff
@@ -17,6 +17,6 @@
 **See**
 
 - [Uses of DocLink](https://github.com/hashicorp/vault/search?l=Handlebars&q=DocLink)
-- [DocLink Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/doc-link.js)
+- [DocLink Source Code](https://github.com/hashicorp/vault/blob/main/ui/app/components/doc-link.js)
```
Comment (stevendpclark) [inline ui/lib/core/stories/doc-link.md:20]: 404

File: ui/lib/core/stories/form-field-groups.md [hunk 3]
```diff
@@ -29,6 +29,6 @@
 **See**
 
 - [Uses of FormFieldGroups](https://github.com/hashicorp/vault/search?l=Handlebars&q=FormFieldGroups)
-- [FormFieldGroups Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/form-field-groups.js)
+- [FormFieldGroups Source Code](https://github.com/hashicorp/vault/blob/main/ui/app/components/form-field-groups.js)
```
Comment (stevendpclark) [inline ui/lib/core/stories/form-field-groups.md:32]: 404

File: ui/lib/core/stories/form-field.md [hunk 4]
```diff
@@ -36,6 +36,6 @@
 **See**
 
 - [Uses of FormField](https://github.com/hashicorp/vault/search?l=Handlebars&q=form-field)
-- [FormField Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/form-field.js)
+- [FormField Source Code](https://github.com/hashicorp/vault/blob/main/ui/app/components/form-field.js)
```
Comment (stevendpclark) [inline ui/lib/core/stories/form-field.md:39]: 404

File: ui/lib/core/stories/layout-loading.md [hunk 5]
```diff
@@ -12,6 +12,6 @@
 **See**
 
 - [Uses of LayoutLoading](https://github.com/hashicorp/vault/search?l=Handlebars&q=LayoutLoading)
-- [LayoutLoading Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/layout-loading.js)
+- [LayoutLoading Source Code](https://github.com/hashicorp/vault/blob/main/ui/app/components/layout-loading.js)
```
Comment (stevendpclark) [inline ui/lib/core/stories/layout-loading.md:15]: 404

File: ui/lib/core/stories/message-error.md [hunk 6]
```diff
@@ -19,6 +19,6 @@
 **See**
 
 - [Uses of MessageError](https://github.com/hashicorp/vault/search?l=Handlebars&q=MessageError)
-- [MessageError Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/message-error.js)
+- [MessageError Source Code](https://github.com/hashicorp/vault/blob/main/ui/app/components/message-error.js)
```
Comment (stevendpclark) [inline ui/lib/core/stories/message-error.md:22]: 404

File: ui/lib/core/stories/popup-menu.md [hunk 7]
```diff
@@ -18,6 +18,6 @@ block.
 **See**
 
 - [Uses of PopupMenu](https://github.com/hashicorp/vault/search?l=Handlebars&q=PopupMenu)
-- [PopupMenu Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/popup-menu.js)
+- [PopupMenu Source Code](https://github.com/hashicorp/vault/blob/main/ui/app/components/popup-menu.js)
```
Comment (stevendpclark) [inline ui/lib/core/stories/popup-menu.md:21]: 404

File: ui/lib/core/stories/select.md [hunk 8]
```diff
@@ -28,6 +28,6 @@ Select components are used to render a dropdown.
 **See**
 
 - [Uses of Select](https://github.com/hashicorp/vault/search?l=Handlebars&q=Select+OR+select)
-- [Select Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/select.js)
+- [Select Source Code](https://github.com/hashicorp/vault/blob/main/ui/app/components/select.js)
```
Comment (stevendpclark) [inline ui/lib/core/stories/select.md:31]: 404

File: ui/lib/core/stories/toolbar/toolbar-download-button.md [hunk 9]
```diff
@@ -17,6 +17,6 @@ It should only be used inside of `Toolbar`.
 **See**
 
 - [Uses of ToolbarDownloadButton](https://github.com/hashicorp/vault/search?l=Handlebars&q=ToolbarDownloadButton+OR+toolbar-download-button)
-- [ToolbarDownloadButton Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/toolbar-download-button.js)
+- [ToolbarDownloadButton Source Code](https://github.com/hashicorp/vault/blob/main/ui/app/components/toolbar-download-button.js)
```
Comment (stevendpclark) [inline ui/lib/core/stories/toolbar/toolbar-download-button.md:20]: 404

File: ui/lib/core/stories/vault-logo-spinner.md [hunk 10]
```diff
@@ -12,6 +12,6 @@
 **See**
 
 - [Uses of VaultLogoSpinner](https://github.com/hashicorp/vault/search?l=Handlebars&q=VaultLogoSpinner)
-- [VaultLogoSpinner Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/vault-logo-spinner.js)
+- [VaultLogoSpinner Source Code](https://github.com/hashicorp/vault/blob/main/ui/app/components/vault-logo-spinner.js)
```
Comment (stevendpclark) [inline ui/lib/core/stories/vault-logo-spinner.md:15]: 404

File: ui/stories/toggle.md [hunk 11]
```diff
@@ -24,6 +24,6 @@ They are a stylistic alternative to checkboxes, but still use the input[type=che
 **See**
 
 - [Uses of Toggle](https://github.com/hashicorp/vault/search?l=Handlebars&q=Toggle+OR+toggle)
-- [Toggle Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/toggle.js)
+- [Toggle Source Code](https://github.com/hashicorp/vault/blob/main/ui/app/components/toggle.js)
```
Comment (stevendpclark) [inline ui/stories/toggle.md:27]: 404

File: website/content/api-docs/relatedtools.mdx [hunk 12]
```diff
@@ -34,4 +34,4 @@ The following list of tools is maintained by the community of Vault users; Hashi
 - [nc-vault-env](https://github.com/namecheap/nc-vault-env) - JS CLI tool that fetches secrets in parallel, puts them into the environment and then `exec`s the process that needs them. Supports auth token renewal, multiple auth backends, verbose logging and dummy mode.
 - [vsh](https://github.com/fishi0x01/vsh) - Interactive shell with tab-completion. Allows recursive operations on paths. Allows migration of secrets between both KV versions.
 
-Want to add your own project, or one that you use? Additions are welcome via [pull requests](https://github.com/hashicorp/vault/blob/master/website/pages/api-docs/relatedtools.mdx).
+Want to add your own project, or one that you use? Additions are welcome via [pull requests](https://github.com/hashicorp/vault/blob/main/website/pages/api-docs/relatedtools.mdx).
```
Comment (stevendpclark) [inline website/content/api-docs/relatedtools.mdx:37]: 404

**Review by stevendpclark — APPROVED** [review]
I've pointed out quite a few 404's that we should probably bring up to someone in docs, but this should still be merged imho.

**Pull request merged**